target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -223,6 +223,9 @@ ramips_board_detect() {
 	*"HC5661A")
 		name="hc5661a"
 		;;
+	*"HC5611")
```

Comment:
drop this to use the generic board detection. The userspace boardname will be `hiwifi,hc5611`.